libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -397,12 +397,16 @@ class MapboxNavigation(
      * Starts listening for location updates and enters an `Active Guidance` state if there's a primary route available
      * or a `Free Drive` state otherwise.
      *
+     * @param withForegroundService Boolean if set to false, [TripService.startService] will not be invoked and
```

Comment:
```suggestion
     * @param withForegroundService Boolean if set to false, foreground service will not be started and
```
nit: `TripService` is an internal API.